sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -1105,23 +1108,21 @@ internal class RadarApiClient(
                     return
                 }
 
-                val customEvent = res.optJSONObject("event")?.let { eventObj ->
+                val conversionEvent = res.optJSONObject("event")?.let { eventObj ->
                     RadarEvent.fromJson(eventObj)
                 }
 
-                val allEvents: MutableList<RadarEvent> = mutableListOf()
-
-                if (trackingEvents != null) {
-                    allEvents.addAll(trackingEvents)
-                }
-
-                if (customEvent != null) {
-                    allEvents.add(0, customEvent)
-                } else {
+                if (conversionEvent == null) {
                     callback.onComplete(RadarStatus.ERROR_SERVER)
+
+                    return
+                } else  {
+                    // The events are returned in the completion handler, but they're also
+                    // sent back via the RadarReceiver.
```

Comment:
I actually don't think we want to send listeners through the event receiver, that was the point of this PR https://github.com/radarlabs/radar-sdk-android/pull/263

basically, customers could get into a loop where they fire a conversion event in the receiver --> event delivered to the receiver --> and so on

Reply:
ah that change went through when this PR was already up. Let's strip this else block then @joeychik-radar

Reply:
im assuming we should make this change on ios too? @jsani-radar

Reply:
yeah, I mirrored the same PR on iOS https://github.com/radarlabs/radar-sdk-ios/pull/218/files